crates/adapters/src/transport/datagen.rs

```diff
@@ -1554,6 +1710,131 @@ transport:
         }
     }
 
+    #[test]
+    #[should_panic]
```

Comment:
I like to specify the message here too

Reply:
good idea, unfortunately the error/panic here is thrown in another thread since we spawn a mock pipeline so I'm not getting it all the way to this test function